Month1/Week2/Section2/Month1-Week2-Section2/Spotify/PlayList.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "Song.h"
+
+class FPlayList {
+
+private:
+	std::string PlayListTitle;
+	const static int MaxSongCount = 30;
+	int SongCount = 0;
+	FSong Songs[MaxSongCount];
+	
+public:
+	FPlayList();
+	FPlayList(std::string NewPlayListTitle);
+	std::string GetListTitle();
```

Comment:
Careful with these kinds of abbreviations. `GetTitle()` and `SetTitle(...)` is more than enough since the context already makes it clear that you mean the playlist title. However, if you really want to be specific, writing `List` instead of `Playlist` reduces clarity and makes the name ambiguous, especially considering what I commented before saying that Playlist is just one word.

Reply:
Modified